### ISSUE TITLE
Fix Pie Chart Gaps

### ISSUE THREAD

**Issue by GeorgeBerdovskiy** [PureChart/purechart#24]
## Background
PureChart now supports simple pie charts thanks to PR #21! However, there's a small rendering issue that results in tiny gaps between slices and some light misalignment.

<img width="1624" alt="image" src="https://github.com/PureChart/purechart/assets/43429820/e7d3b0dc-bf52-4a7a-9085-58981745f865">

## Goal
Modify pie chart rendering logic to eliminate small gaps and misalignment - it should look perfect! One potential culprit is rounding errors, so you may want to start there. Good luck!

**Comment by GeorgeBerdovskiy** [comment]
Note - the data should be formatted like this.
```rb
[
  {
    value: 25,
    color: "#eb4d4b"
  },
  {
    value: 100,
    color: "#be2edd"
  },
  ...
]
```

**Comment by skyz04** [comment]
Ill take a crack at this

**Comment by GeorgeBerdovskiy** [comment]
Sorry @skyz04, Ian basically fixed this in #31 when he switched to path rendering. I've reserved #30 for you though! Let me know if you'd like to do something else and I'll take you off that issue.
